1080i/Includes_Info.xml

```diff
@@ -1605,12 +1605,12 @@
                         <param name="dbtype_05">video</param>
                     </include>
                     <param name="items_03">true</param>
```

Comment:
Switch to `items_02` as the third label will always be empty and we don't need it.